templates/components/checkbox.build.hbs

```diff
@@ -1,6 +1,29 @@
 <template>
   <div v-if="description" class="help-block description">\{{ description }}</div>
 
+   <template v-if="selectAll">
+    <div class="checkbox checkbox-icon">
+      <input
+        type="checkbox"
+        :id="name + '-' + 'selectAll'"
+        :name="name"
+        v-model="checkValue"
+        value="Select All"
+        tabindex="-1"
+      >
+      <label v-on:click="checkAllHandler()">
+        <span
+          class="check focus-outline"
+          tabindex="0"
+          v-on:keydown.space.prevent="checkAllHandler()"
+          >
+            <i class="fa fa-check"></i>
+        </span>
+        <span class="option-item">\{{isSelectAll[0].label}}</span>
```

Comment:
Comment not addressed https://github.com/Fliplet/fliplet-widget-form-builder/pull/451/files#r923399173